src/components/ResultsList.vue

```diff
@@ -1,23 +1,20 @@
 <template>
   <section class='results-list'>
-    <h1>Result List</h1>
-    <section :key='i' v-for='(result, i) in searchResults'>
+    <h4 v-if="!$store.state.searchResults.length">
```

Comment:
This is good error handling. We will want a setTimeout here as well to indicate that our API data is loading.